tests/qemu/qemu.pm

```diff
@@ -49,7 +49,8 @@ sub run {
     }
     elsif (is_ppc64le) {
         is_qemu_preinstalled or install_qemu('qemu-ppc');
-        enter_cmd "qemu-system-ppc64 -nographic";
+        record_soft_failure('workaround for bsc#1230042 - [SLM6.1][PPC64LE] Fail to start QEMU - qemu-system-ppc64');
+        enter_cmd "qemu-system-ppc64 -vga none -nographic";
```

Comment:
PPC test will end up on PowerVM, in the long run. @guoxuguang  Can you add a comment to explain that above line is for openqa test running on qemu emulated ppc vm(starting from SLM 6.1), and we are going to recover `-vga none` once this way is abandoned?

Reply:
Yes, PPC images test will end up on PowerVM as our expected. However, depend on current situation, we have to boot up ppc images via QEMU without KVM. 
Meanwhile, we are going to recover `-vga none` once PowerVM is available for our openqa PPC image test. 
Add the new comment - `workaround for bsc#1230042 - plan to remove this workaround until PPC images test on PowerVM `